### PR TITLE
blog: CCS wallet incident

### PR DESCRIPTION
**Pull request by plowsof**
after some discussion, a simple summary that points to the meta issue containing all the details has been agreed upon. i have pushed a draft here.

copy of https://github.com/monero-project/meta/issues/916

### CODE REVIEW

**Review by rbrunner7 — APPROVED** [review]
I checked and can confirm that the PR does what the  PR claims: It adds a blog post to the getmonero.org blog about this incident, as a copy of Luigi's announcement on GitHub.

What I can't judge of course: Whether making such a blog post in this form is a good idea in the first place, and whether people agree to do so.

My personal opinion: Looks ok to do so, because there will be more communication to come.

**Review by erciccione — COMMENTED** [review]
Not sure if core wants to send the message sent on github as a blog post. Better ask them what plans they have first. If this is good to go with them, we can quickly merge and publish it.

**Review by erciccione — APPROVED** [review]

File: _posts/2023-11-03-ccs-wallet-incident.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: CCS Wallet Incident
+summary: 2,675.73 XMR (the entire balance) was drained from the CCS wallet on September 1, 2023. The General Fund wallet remains intact.
+tags: [urgent, announcements, core, community]
+author: Monero Core Team
+---
+
+{% t global.lang_tag %}
```
Comment (erciccione) [inline _posts/2023-11-03-ccs-wallet-incident.md:9]: This is not used in blog posts.

File: _posts/2023-11-03-ccs-wallet-incident.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: CCS Wallet Incident
+summary: 2,675.73 XMR (the entire balance) was drained from the CCS wallet on September 1, 2023. The General Fund wallet remains intact.
+tags: [urgent, announcements, core, community]
```
Comment (erciccione) [inline _posts/2023-11-03-ccs-wallet-incident.md:5]: I would also remove the community tag here

**Comment by plowsof** [comment]
after discussion, a simple explination, pointing to the meta which contains the full detail was agreed upon, i have pushed a draft of this

**Review by erciccione — APPROVED** [review]

File: _posts/2023-11-03-ccs-wallet-incident.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: Community Crowdfunding System wallet incident
+summary: 2,675.73 XMR (the entire balance) was drained from the CCS wallet on September 1, 2023. The General Fund wallet remains intact.
+tags: [urgent, announcements, core]
+author: Monero Core Team
+---
+
+During a routine transfer, we noticed the '[Community Crowdfunding System](https://ccs.getmonero.org/)' (CCS) main Wallet was drained of 2,675.73 XMR (the entire balance) on September 1, 2023. The hot wallet, used for payments to contributors, is untouched; its balance is ~244 XMR. We have thus far not been able to ascertain the source of the breach. An internal investigation is ongoing and further details will be released once we have them.
```
Comment (nahuhh) [inline _posts/2023-11-03-ccs-wallet-incident.md:9]:
```suggestion
During a routine transfer, we noticed the '[Community Crowdfunding System](https://ccs.getmonero.org/)' (CCS) Donation Wallet was drained of 2,675.73 XMR (the entire balance) on September 1, 2023. A second wallet, used for payments to proposers, is untouched; its balance is ~244 XMR. We have thus far not been able to ascertain the source of the breach. An internal investigation is ongoing and further details will be released once we have them.
```

Comment (nahuhh) [reply _posts/2023-11-03-ccs-wallet-incident.md:9]: main Wallet mixed capitalization
clarification that the hot wallet is not the same wallet mentioned previously

Comment (HardenedSteel) [reply _posts/2023-11-03-ccs-wallet-incident.md:9]: What about using cold and hot wallet terms?

Comment (nahuhh) [reply _posts/2023-11-03-ccs-wallet-incident.md:9]: They're both hot, technically.

Comment (plowsof) [reply _posts/2023-11-03-ccs-wallet-incident.md:9]: proposers become contributors, then they receive moneys. both wallets are hot. there was a main + secondary wallet. 

replacing "hot" with "secondary" + fixing mixed capitalisation should be OK

**Review by erciccione — APPROVED** [review]

**Review by nahuhh — APPROVED** [review]

**Review by HardenedSteel — APPROVED** [review]

**Pull request merged**